tools/cs_profile_template.yaml

```diff
@@ -0,0 +1,12 @@
+table:
+
+table_definition:
+
+keyspace:
+
+keyspace_definition:
+
+queries:
+  test_query:
+    cql:
+    fields: samerow
```

Comment:
`samerow` or `somerow` ?

Reply:
I (and the future readers even more) would appreciate comments in the template explaining the semantics and what is expected where.

Reply:
`samerow`, see the examples in c-s: https://cassandra.apache.org/doc/stable/cassandra/tools/cassandra_stress.html#profile. IIRC, `queries.<some_query>.fields` parameter is required and accepts either `samerow` or `multirow`. We don't support this parameter in cql-stress yet. From what I remember, this is somehow related to batches (`multirow` would then insert multiple rows in one batch statement). I don't think we should focus on this parameter right now, as it's not commonly used (and AFAIK, the batches are discouraged anyway).

Apart from that, this file is not to be merged anyway. See my other comment where I mention why the file was removed.